sherpa/ui/utils.py

```diff
@@ -14454,26 +14647,50 @@ def plot(self,
         Draw the data and residuals for the default dataset and then
         overplot those from dataset 2:
 
-        >>> plot("data", "resid", cols=1, color="black")
+        >>> plot("data", "resid", color="black")
         >>> plot("data", 2, "resid", 2, overplot=True, color="black", alpha=0.5)
 
+        Draw the data and residuals for the default dataset and then
+        overplot those from dataset 2:
+
+        >>> plot("data", "resid", ids=[1, 2], color="black", alpha=[1, 0.5])
```

Comment:
```suggestion
        An alternative way to generate the same output is to call ``plot`` once
        and pass in both the default it (1 in this case) and the id for the
        second dataset:

        >>> plot("data", "resid", ids=[1, 2], color="black", alpha=[1, 0.5])
```

Reply:
So, the way we create `ahelp` versions of these files is to grab these examples and split them by "text followed by code" is a single example - e.g. https://cxc.cfa.harvard.edu/sherpa/ahelp/plot.html#examples . This means that ahelp users don't quite get the same context (e.g. as written in your suggestion). I'll try to rewrite this